msgraph-sdk-java.yml

```diff
@@ -0,0 +1,38 @@
+page_type: sample
+products:
+  - office-excel
+  - office-365
+  - office-outlook
+  - office-onedrive
+  - office-onenote
+  - office-planner
+  - office-sp
+  - ms-graph
+languages:
+  - java
+extensions:
+  contentType: sdks
+  technologies:
+    - Microsoft Graph
+    - Microsoft identity platform
+  services:
+    - Education
+    - Excel
+    - Groups
+    - Insights
+    - Intune
+    - Microsoft identity platform
+    - Office 365
+    - Outlook
+    - OneDrive
+    - OneNote
+    - Planner
+    - Reports
+    - Security
+    - SharePoint
+    - Users
+  platforms:
+    - Android
+  createdDate: '9/12/2017 10:20:15 AM'
+title: Microsoft Graph SDK for Java
+description: Microsoft Graph SDK for Java
```

Comment:
Microsoft Graph SDK for Java and Android.

Reply:
Updated!